modules/metrics-probe/metrics-probe.c

```diff
@@ -153,7 +195,8 @@ _process(LogParser *s, LogMessage **pmsg, const LogPathOptions *path_options, co
             evt_tag_msg_reference(*pmsg));
 
   StatsCounterItem *counter = _lookup_stats_counter(self, *pmsg);
-  stats_counter_inc(counter);
+  gint64 increment = _calculate_increment(self, *pmsg);
```

Comment:
`gint64` and `gssize` are not always the same.
I think we should use  `gssize` here, which is the base type of stats counters.

Reply:
`gssize` + `strtol()` would be the best right?

Reply:
I think gssize and strtoll would be the best, and we could let it overflow on 32-bit.

(Or scanf() with the gssize formatter, but that's expensive.)